config.py

```diff
@@ -209,14 +208,14 @@
 
 # Regionalization
 paths["region"] = root + "03 Intermediate files" + fs + "Files " + region + fs
-if not os.path.isdir(paths["Region"]):
-    os.mkdir(paths["Region"])
+if not os.path.isdir(paths["region"]):
+    os.makedirs(paths["region"] + "Renewable energy" + fs)
```

Comment:
makedirs vs mkdir: makedirs create the whole folder structure, not just one folder. To be implemented in generate models later...